client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx

```diff
@@ -208,7 +216,7 @@ const DomainsStep: Step = function DomainsStep( { navigation, flow } ) {
 			return __( 'Your domain. Your identity.' );
 		}
 
-		if ( flow === HUNDRED_YEAR_PLAN_FLOW ) {
+		if ( flow === HUNDRED_YEAR_PLAN_FLOW || flow === HUNDRED_YEAR_DOMAIN_FLOW ) {
```

Comment:
Just a nit: Lower down a different approach is used:
```
[ HUNDRED_YEAR_PLAN_FLOW, HUNDRED_YEAR_DOMAIN_FLOW ].includes( flow)
```

Each of these could also be simplified to
```
const HUNDRED_YEAR_FLOWS = [ HUNDRED_YEAR_PLAN_FLOW, HUNDRED_YEAR_DOMAIN_FLOW ]; // at the top
HUNDRED_YEAR_FLOWS.includes( flow )
```

Reply:
I had the same thought 😬 in this specific case, the `if` block right above this one uses the same style (`===` instead of `[].includes`), so I was in doubt if I should use the array comparison or not. I'll update this to use the array one to be consistent with the other updates 🙂